src/App.js

```diff
@@ -102,6 +102,24 @@ export default function App() {
         setOutput(translation["response"])
     }
 
+    /**
+     * Function accesses google search custom bar to add query from input 
+     * text box directly into search bar which then searches google. 
+     */
+    function querySearch(inputValue, outputLanguageValue) {
+      let query = inputValue + " in "
+      if (outputLanguageValue === "js") { // FIXME: this should eventually come from backend
+        query += "javascript";
+      }
+      if (outputLanguageValue === "py") {
+        query += "python";
+      }
+
+      document.getElementById("gsc-i-id1").value = query;
```

Comment:
Have you looked into using refs?

Reply:
No - I haven't. I can start looking into them now if you think they'd be useful. Would appreciate any resources! Also will add a TODO

Reply:
👍 [This](https://medium.com/@rossbulat/react-using-refs-with-the-useref-hook-884ed25b5c29) seems like a decent (and extremely detailed) guide.